
syntax = "proto3";

package delivery.user;

service UserLogin {
    rpc Login(LoginRequest) returns(LoginResponse);
}

message LoginRequest {
    string username = 1;
    string password = 2;
}

message LoginResponse {
    string token = 1;
}
